making-htap-database-reality-what-i-learned-from-pingcap-vldb-paper.md

```diff
@@ -0,0 +1,106 @@
+---
+title: 'Making an HTAP Database a Reality: What I learned from PingCAP’s VLDB paper'
+author: ['Xianlin Chen']
+date: 2020-10-22
+summary: 'Recently, VLDB published the PingCAP paper, TiDB: A Raft-based HTAP Database. This article, a DBA at PalFish shares his thoughts on the article and his expectations for TiDB’s future development.'
+tags: ['HTAP', 'VLDB', 'Consistency', 'Raft']
+categories: ['HTAP']
+image: /images/blog/making-htap-database-reality-what-i-learned-from-pingcap-vldb-paper.jpg
+---
+
+**Author:** Xianlin Chen (DBA at PalFish)
+
+![banner](media/making-htap-database-reality-what-i-learned-from-pingcap-vldb-paper.jpg)
```

Comment:
```suggestion
![HTAP database](media/making-htap-database-reality-what-i-learned-from-pingcap-vldb-paper.jpg)
```

Reply:
Next time, you can use a shorter image name with SEO keywords.

Reply:
Descriptive ALT text with an SEO keyword helps our SEO performance, so I want to use "HTAP database."